events-api/src/modules/auth/dto/requests/login.dto.ts

```diff
@@ -1,6 +1,8 @@
+import { ApiProperty } from '@nestjs/swagger';
 import { IsEmail, IsString, MinLength, MaxLength } from 'class-validator';
 
 export class LoginDto {
+  @ApiProperty({ example: 'gordon@freeman.com' })
```

Comment:
You can use format: email to tell swagger that it is an email

Reply:
> You can use format: email to tell swagger that it is an email

fixed